### PR TITLE
Upgrades returns Task()

### PR DESCRIPTION
**Pull request by danielui**
# Pull Request summary:
- Upgrades returns Task instance instead task_id
- Update README

# Checklist:
- [X] Make sure to run pre-commit before commiting changes
- [X] Make sure all checks have passed
- [X] PR description is clear and comprehensive
- [X] Mentioned the issue that this PR solves (if applicable)
- [X] Make sure you test the changes

### CODE REVIEW

**Review by tehAgitto — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -57,20 +57,19 @@ speedtest = session.api.speedtest.speedtest(devices[0], devices[1])
...
 session.api.repository.upload_image(software_image)
 
-# Upgrade
-software_action = SoftwareActionAPI(session, DeviceCategory.VEDGES)
-software_action_id = software_action.upgrade_software(vsmarts,
-    InstallSpecHelper.CEDGE.value, reboot = False, sync = True, software_image=software_image)
+# Install software
+
+install_task = session.api.software.install(devices = vsmarts,
+    software_image= software_image)
```
Comment (sbasan) [inline README.md:69]: can wo go for shorter "image" instead "software_image" for keyword argument here ?

**Review by sbasan — COMMENTED** [review]
lgtm, added one suggestion

File: README.md (hunk 1 above)
Comment (danielui) [reply README.md]: Sure, done

Comment (sbasan) [reply README.md]: i meant the KEYWORD ARGUMENT name not variable name in example :)

File: vmngclient/api/software_action_api.py [hunk 2]
```diff
@@ -92,7 +93,7 @@ def install(
         sync: bool = True,
         software_image: Optional[str] = "",
```
Comment (sbasan) [inline vmngclient/api/software_action_api.py:94]: can we go shorter name "image" ?

File: README.md (hunk 1 above)
Comment (danielui) [reply README.md]: Okey, it's make more sense

**Review by sbasan — APPROVED** [review]
lgtm

**Review by tomaszwarda — APPROVED** [review]

**Pull request merged**